src/main/java/org/komamitsu/fluency/buffer/Buffer.java

```diff
@@ -50,7 +50,7 @@ public void init()
                     @Override
                     public void process(List<String> params, FileChannel channel)
                     {
-                        LOG.info("Loading buffer: params={}, buffer={}", params, channel);
+                        LOG.debug("Loading buffer: params={}, buffer={}", params, channel);
```

Comment:
This log message is a bit important and appears only when file backup mode is enables, I want to leave it `info`.